portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -714,375 +836,441 @@ definition {
+		}
 
-		LAR.exportSiteCP(siteScopeName = "Site Name");
+		task ("And: Exports the site") {
+			LAR.exportSiteCP(siteScopeName = "Site Name");
 
-		var key_larFileName = "${larFileName}";
+			var key_larFileName = "${larFileName}";
 
-		AssertTextEquals(
-			locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS_1",
-			value1 = "Successful");
+			AssertTextEquals(
+				locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS_1",
+				value1 = "Successful");
 
-		var larFileName = LAR.getLarFileName();
+			var larFileName = LAR.getLarFileName();
 
-		LAR.downloadLar();
+			LAR.downloadLar();
+		}
 
-		JSONGroup.addGroup(groupName = "Site Name B");
+		task ("Then: User can import the LAR file on a new site without error on the console") {
+			JSONGroup.addGroup(groupName = "Site Name B");
 
-		LAR.importSiteCP(
-			larFileName = "${larFileName}",
-			siteName = "Site Name B");
+			LAR.importSiteCP(
+				larFileName = "${larFileName}",
+				siteName = "Site Name B");
 
-		AssertConsoleTextNotPresent(value1 = "java.lang.NullPointerException");
+			AssertConsoleTextNotPresent(value1 = "java.lang.NullPointerException");
+		}
 	}
 
 	@description = "Verify is possible export/import a Portlet."
 	@priority = "5"
 	test ExportImportPortlet {
 		property portal.acceptance = "true";
 
-		JSONLayout.addPublicLayout(
-			groupName = "Guest",
-			layoutName = "Test Page Name");
+		task ("Given: User adds a new page with an IFrame portlet") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Test Page Name");
 
-		Navigator.gotoPage(pageName = "Test Page Name");
+			Navigator.gotoPage(pageName = "Test Page Name");
 
-		Portlet.addPG(portletName = "IFrame");
+			Portlet.addPG(portletName = "IFrame");
+		}
 
-		Portlet.gotoPortletOptions(
-			portletName = "IFrame",
-			portletOption = "Look and Feel Configuration");
+		task ("And: Configure a custom look for the portlet") {
+			Portlet.gotoPortletOptions(
+				portletName = "IFrame",
+				portletOption = "Look and Feel Configuration");
 
-		IFrame.selectConfigurationFrame();
+			IFrame.selectConfigurationFrame();
 
-		Portlet.configureGeneralViaLookAndFeelPG(customTitle = "IFrame Custom");
+			Portlet.configureGeneralViaLookAndFeelPG(customTitle = "IFrame Custom");
 
-		IFrame.closeFrame();
+			IFrame.closeFrame();
+		}
 
-		LAR.exportPortlet(larFileName = "IFrame.lar");
+		task ("When: User exports the portlet then create a new site with a page") {
+			LAR.exportPortlet(larFileName = "IFrame.lar");
 
-		JSONGroup.addGroup(groupName = "Test Site Name");
+			JSONGroup.addGroup(groupName = "Test Site Name");
 
-		JSONLayout.addPublicLayout(
-			groupName = "Test Site Name",
-			layoutName = "Test Site Page");
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Site Page");
+		}
 
-		Navigator.gotoSitePage(
-			pageName = "Test Site Page",
-			siteName = "Test Site Name");
+		task ("Then: User can create a similar portlet, then import the LAR file") {
```

Comment:
It's more reasonable to change this "Then" clause to "And" clause so that make it as an action instead of a result.
Example:
"And: Add an IFrame portlet to the page, then import the previous LAR file"